### DISCUSSION TITLE
Recover an extension after a crash

### DISCUSSION THREAD

**Discussion by DorianHawkmoon** [Ulauncher/Ulauncher#1223] [Q&A]
So testing Ulauncher, I installed a small extension (specifically Session Manager), and lacking the support for openbox, I decided to add a line in main.py. Of course it failed and I triggered a crash.

My question is, once the extension has crashed, how can I restart it?
I mean, once crashed, it no longer works, but even after a reboot, the extension is still under a crash state, and I don't think uninstalling and installing again is the correct way.

**Comment by friday** [comment]
The easiest way now is to remove/reinstall the extension or restart Ulauncher. We may have an issue for this somewhere. There's several missing small UI features like this.
